src/homework/lesson1/RestaurantTableReservation.java

```diff
@@ -0,0 +1,88 @@
+package homework.lesson1;
+
+import java.util.Scanner;
+
+public class RestaurantTableReservation {
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        boolean[] tablesAvailability = {true, false, true, false, false};
+
+        int userChoice = getUserChoice();
+        switch (userChoice) {
+            case 1:
+                makeReservation(tablesAvailability);
+                break;
+            case 2:
+                cancelReservation(tablesAvailability);
+                break;
+            case 3:
+                System.out.println("Thank you for using the table reservation system!");
+                return;
+            default:
+                System.out.println("Wrong choice. Please try again.");
+                getUserChoice();
+        }
+
+        scanner.close();
+    }
+
+    private static int getUserChoice() {
+        int input;
+
+        while (true) {
+            System.out.println("\nReservation Menu:");
+            System.out.println("1. Reserve a table");
+            System.out.println("2. Cancel reservation");
+            System.out.println("3. Exit");
+            System.out.print("\nPlease enter your choice: ");
+            input = scanner.nextInt();
+            if (input >= 1 && input <= 3) {
+                break;
```

Comment:
instead of break you can use return input right here

Reply:
Got it, thanks. I'll make the changes in separate commit and add it to the master branch